PcBdsPkg/Include/Library/MsBootPolicyLib.h

```diff
@@ -15,7 +15,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
  * application is at PcsBdsPkg/MsBootPolicy.
  *
  * Boot Policy Applications can be created to meet a platform's requirements. The PcBdsPkg
- * example aplication impelements USB, PXE and HDD boot sequences, in addition to a default
+ * example application impelements USB, PXE and HDD boot sequences, in addition to a default

```

Comment:
SpellCheck is in AuditOnly mode for PcBdsPkg. I don't have a problem with this spell fix here, but also probably worthwhile to turn off AuditOnly mode and fix all the spelling errors in a separate patch. Up to you.

Reply:
Sounds good. Will revert for now, and worry about it in a separate PR.